test/JDBC/upgrade/master/schedule

```diff
@@ -333,6 +333,7 @@ TestVariableDataLength
 TestXML
 timefromparts
 triggers_with_transaction
+typeid-typename
```

Comment:
Why are we not adding typeid-typename-dep here?

Reply:
Dump restore check was failing due to logging in from different user, so I confirmed it with rishabh to remove it.